HARK/abstract/tests/consindshk.yml

```diff
@@ -0,0 +1,39 @@
+---
+states: !StateSpace
+  variables:
+    - !State
+      name: m
+      short_name: money
+      long_name: market resources
+      latex_repr: \mNrm
```

Comment:
I know how you're imagining using the Latex representation.
But my preference would be to not include it in the PR unless you have some demonstration of how it works ready.
4 different ways to name something for a quick demo seems like a lot....

Reply:
For now these are filler, I want to throw any non-required keys into an attributes dictionary.